client/map.js

```diff
@@ -0,0 +1,23 @@
+var drawMap = function(coordinates){
```

Comment:
Please place a single space after the params in functions
e.g. 
var drawMap = function(coordinates) {